docs/src/pages/[platform]/connected-components/authenticator/configuration/loginMechanisms.react-native.mdx

```diff
@@ -0,0 +1,38 @@
+import { Alert, Authenticator } from '@aws-amplify/ui-react';
```

Comment:
Due to how web and flutter are split, react-native had to be its own file. We should look into reducing these duplications, but for the time being, this should suffice.